tech/languages/dotnet/dotnet-install.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Install .NET Core
+subsection: dotnet
+section: tech-languages
+order: 1
+version: 7.8.4
```

Comment:
Not sure what version refers to. Kept it at the same value as the about page had. Maybe this needs a change.